modules/tile-converter/src/i3s-converter/i3s-converter.js

```diff
@@ -165,7 +165,7 @@ export default class I3SConverter {
         href: './1',
         ...coordinates
       });
-      const child = await this._createNode(root0, sourceRootTile, parentId, 0);
+      const [child] = await this._createNode(root0, sourceRootTile, parentId, 0);
```

Comment:
Recommend adding JSDoc types to `_createNode` so that we get automatic type check for this desctructuing.